addons/api/tests/unit/serializers/auth-method-test.js

```diff
@@ -271,4 +429,68 @@ module('Unit | Serializer | auth method', function (hooks) {
     assert.strictEqual(record.issuer, issuer);
     assert.strictEqual(record.max_age, maxAge);
   });
+
+  test('it normalizes LDAP records', async function (assert) {
+    const store = this.owner.lookup('service:store');
+
+    const ldapUrl = 'ldap://ldap.example.com';
+    const certificates = ['certificate-1234', 'certificate-5678'];
+    const clientCerticate = 'certificate-1234';
+    const startTls = true;
+    const insecureTls = false;
+    this.server.get('/v1/auth-methods/ldap123', () => ({
+      attributes: {
+        account_attribute_maps: ['from=to', 'foo=bar'],
+        urls: [ldapUrl],
+        start_tls: startTls,
+        insecure_tls: insecureTls,
+        discover_dn: true,
+        anon_group_search: true,
+        bind_dn: 'cn=users,dc=example,dc=com',
+        upn_domain: 'example.com',
+        user_dn: 'cn=users,dc=example,dc=com',
+        user_attr: 'uid',
+        user_filter: '({{.UserAttr}}={{.Username}})',
+        enable_groups: true,
+        group_dn: 'cn=groups,dc=example,dc=com',
+        group_attr: 'cn',
+        group_filter: '(member={{.UserDN}})',
+        certificates: certificates,
+        client_certificate: clientCerticate,
+      },
```

Comment:
Non-blocking question: to test normalization works shouldn't the `client_certificate_key` be added here and then we test that it gets set to an empty string? Im probably misunderstanding so this is more of a question for understanding.

Reply:
Yup we should have a test for normalization that confirms it sets the secrets to an empty string correctly

Reply:
@ZedLi @lisbet-alvarez changes incoming